resources/dicts/relationship_events/normal_interactions/dislike/increase.json

```diff
@@ -1,571 +1,435 @@
+			"vengeful",
+			"bloodthirsty"
+		]
+	},
+	{
+		"id": "dislike_inc_med5",
+		"interactions": [
+			"r_c is asking m_c to tell {PRONOUN/r_c/object} about how good {PRONOUN/r_c/subject} {VERB/m_c/look/looks}.",
+			"r_c offends m_c with {PRONOUN/r_c/poss} brutal honesty.",
+			"m_c is irritated by r_c's constant bragging.",
+			"m_c can't believe r_c just said that!",
+			"m_c tells r_c to quiet down, but r_c doesn't listen.",
+			"r_c is boasting about {PRONOUN/r_c/poss} accomplishments to m_c, but m_c isn't impressed."
+		],
+		"random_trait_constraint": [
+			"shameless",
+			"bold"
+		]
+	},
+	{
+		"id": "dislike_inc_med6",
+		"interactions": [
+			"r_c pulled a prank on m_c.",
+			"r_c blamed m_c for {PRONOUN/r_c/poss} own mistake.",
+			"m_c is tired of all the pranks r_c pulls on {PRONOUN/m_c/object}.",
+			"m_c notices that things always go wrong when r_c is around.",
+			"m_c starts to always blame r_c if something bad happens.",
+			"m_c thinks r_c is bad luck.",
+			"r_c won't stop bothering m_c."
+		],
+		"random_trait_constraint": [
+			"troublesome"
+		]
+	},
+	{
+		"id": "dislike_effect_other1",
+		"relationship_constraint": [
+			"dislike_40"
+		],
+		"interactions": [
+			"r_c is spreading a rumor about m_c.",
+			"r_c is mocking m_c.",
+			"r_c is bragging about being better than m_c.",
+			"r_c is belittling m_c's accomplishments.",
+			"r_c is always talking about how much {PRONOUN/r_c/subject} {VERB/r_c/dislike/dislikes} m_c.",
+			"r_c's friends are tired of hearing about all of the things {PRONOUN/r_c/subject} {VERB/r_c/hate/hates} about m_c.",
+			"The Clan is starting to get nervous about how much m_c and r_c dislike one another.",
+			"m_c and r_c argue and fight like it's in their blood to do so.",
+			"The Clan worries about whether r_c would help m_c if something went wrong.",
+			"r_c is telling others about how terrible m_c can really be.",
+			"r_c leaves as soon as m_c arrives and refuses to be nearby."
+		]
+	},
+	{
+		"id": "dislike_kit_inc_med1",
+		"interactions": [
+			"m_c trips over r_c.",
+			"m_c had to nip r_c on the rump because {PRONOUN/r_c/subject} {VERB/r_c/were/was} being naughty.",
+			"m_c growls at r_c to get out from under {PRONOUN/m_c/poss} paws.",
+			"m_c snaps at r_c's annoying antics.",
+			"m_c tells someone to get r_c away from {PRONOUN/m_c/object}."
+		],
+		"main_status_constraint": [
+			"elder",
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"kitten"
+		],
+		"reaction_random_cat": {
+			"dislike": "increase"
+		}
+	},
+	{
+		"id": "dislike_dep_leader_inc_med1",
+		"interactions": [
+			"m_c punishes r_c with extra work.",
+			"m_c divides r_c into extra patrols.",
+			"m_c gives r_c a particularly hard task, expecting {PRONOUN/r_c/object} to fail.",
+			"m_c threatens r_c with never passing {PRONOUN/r_c/poss} assessment!",

```

Comment:
Needs to be in an apprentice specific interaction, won't make sense for a warrior.